cli/candidate_test.go

```diff
@@ -22,6 +22,7 @@ func TestRegisterCandidate(t *testing.T) {
 		"--rpc-endpoint", "http://"+e.RPC.Addr,
 		"--wallet", validatorWallet,
 		"--from", validatorAddr,
+		"--force",
```

Comment:
We need to add this flag to `register` command, currently, it's in the test only.

Reply:
This flag is for `multitransfer`, not `register`.